packages/manager/apps/dedicated/client/app/dedicated/server/display-name/display-name.controller.js

```diff
@@ -61,8 +56,16 @@ angular.module('App').controller(
             ].join('. '),
             'server_dashboard_alert',
           );
-          this.$state.go('^');
         })
+        .finally(() =>
+          this.$state.go(
+            '^',
+            {},
+            {
+              reload,
+            },
+          ),
+        )
         .finally(() => {
           this.loading.update = false;
         });
```

Comment:
Maybe we can merge the 2 .finally()?

Reply:
The second `finally` depends on the promise returned by `$state.go`. So they both can not be combined.